contracts/examples/NFTExample.sol

```diff
@@ -38,29 +35,17 @@ contract NFTExample is
             super.supportsInterface(interfaceId);
     }
 
-    function setTokenURI(
-        uint256 tokenId,
-        ctUint64 _itTokenURI,
-        bytes calldata _itSignature
-    ) public {
-        _requireOwned(tokenId);
-        address owner = _ownerOf(tokenId);
-        if (msg.sender != owner) {
-            revert ERC721IncorrectOwner(msg.sender, tokenId, owner);
-        }
-
-        itUint64 memory it;
-        it.ciphertext = _itTokenURI;
-        it.signature = _itSignature;
-
-        _setTokenURI(owner, tokenId, MpcCore.validateCiphertext(it));
-    }
-
-    function mint(address to) public onlyOwner {
+    function mint(
```

Comment:
since this NFT only allows private URI, should we change the file name also to maybe PrivateNFTExample.sol?

Reply:
We can